benches/basic.rs

```diff
@@ -25,19 +27,18 @@ pub fn ispc_downsampler(c: &mut Criterion) {
 
 pub fn resize_rs(c: &mut Criterion) {
     if let LoadResult::ImageU8(img) = load(Path::new("test_assets/square_test.png")) {
-        let target_width = img.width / 4;
-        let target_height = img.height / 4;
+        let target_width = img.width / DOWNSCALE;
+        let target_height = img.height / DOWNSCALE;
 
         let src = img
             .data
             .chunks(3)
             .map(|v| RGB::new(v[0], v[1], v[2]))
             .collect::<Vec<_>>();
 
-        let mut dst = vec![RGB::new(0, 0, 0); target_width * target_height];
-
         c.bench_function("Downsample `square_test.png` using resize", |b| {
             b.iter(|| {
+                let mut dst = vec![RGB::new(0, 0, 0); target_width * target_height];
```

Comment:
Don't really want to benchmark allocation as part of the resize test? Maybe not even instantiating the resizer (which we might want to benchmark separately as it iirc pre-computes weight tables?).